Models/DTOs/CustomerCreate.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PSP.Models.DTOs
+{
+    public class CustomerCreate
+    {
+        public string Username { get; set; }
+        public string Password { get; set; }
+        public int LoyaltyPoints { get; set; }
+    }
```

Comment:
Sure, but then we would have to make admin authentication for this, right?

Reply:
not sure what you mean by that exactly - you mean we should add authorization to check if role is admin for the create method? i could add that with my next pr if thats the problem

Reply:
added the role. What I'm saying is this endpoint probably won't need authorization, so be careful not to make it so that anyone can create an admin user